__tests__/ReactRelayFragmentContainer-test.tsx

```diff
@@ -61,9 +69,10 @@ describe('ReactRelayFragmentContainer', () => {
     let spec;
     let variables;
 
-    class ContextSetter extends React.Component {
+    class ContextSetter extends React.Component<any, any> {
+        __relayContext: { environment: any };
```

Comment:
Can we use types from `@types/relay-runtime`?

Reply:
`@types/relay-runtime` causes problems in the core library because internal api are used